Cython/Compiler/Code.py

```diff
@@ -450,6 +450,9 @@ def format_code(self, code_string, replace_empty_lines=re.compile(r'\n\n+').sub)
     def __str__(self):
         return "<%s(%s)>" % (type(self).__name__, self.name)
 
+    def __repr__(self):
+        return str(self)
+
```

Comment:
Why did you add this? Just for better debugging? I would expect `repr()` to return something else than what `str()` currently returns here, something more useful for coding. Not sure what that could be, though.

That said, since `str()` falls back to `repr()` but not the other way round, I'd rather rename `__str__` to `__repr__` above, if we want to keep both the same.

Reply:
Yes just for debugging